js/src/lib/Search/index.js

```diff
@@ -43,6 +43,7 @@ const Search = props => (
       ]}
     />
     <SearchBox
+      autoFocus={window.location.pathname.includes('/packages/')}
```

Comment:
Wow, it's pretty cool that you have a test env for every PR. Love that.

Thanks for the review. I agree that we should drop the trailing slash and it should be `window.location.pathname.includes('/packages')`, but not `/package`. I don't think the correct behavior should be to highlight the search bar when you load a [search result][result] - at least that's not what I would expect. Having `/packages` would limit it to only the sarch screen.

Agree?

[result]: https://yarnpkg.com/en/package/babel